Makefile

```diff
@@ -334,5 +334,5 @@ CABS_MSI = "*.cab,crc-windows-amd64.msi"
 $(BUILD_DIR)/windows-amd64/crc-windows-installer.zip: $(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi
 	rm -f $(HOST_BUILD_DIR)/crc.exe
 	rm -f $(HOST_BUILD_DIR)/crc-embedder
-	rm -f $(HOST_BUILD_DIR)/split
+	rm -f $(HOST_BUILD_DIR)/GenMsiWxs
```

Comment:
@anjannath I don't have issue with tool name `GenMsiWxs` but the file name should be `gen_msi_wxs.go` , In go variable name should follow mixedCaps not the filenames.

Reply:
Right, will change the name to `gen_msi_wxs.go`, thanks